.github/workflows/awsfulltest.yml

```diff
@@ -3,6 +3,8 @@ name: nf-core AWS full size tests
 # It runs the -profile 'test_full' on AWS batch
 
 on:
+  branches:
+    - master
```

Comment:
```suggestion
  workflow_dispatch:
```

Reply:
If you would like to test the full tests, one can activate this workflow for workflow dispatch events as well

Reply:
sth. missing after the dispatch?